tests/test_cp_pipe.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+#
+# Copyright 2008-2017  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+"""Test cases for cp_pipe."""
+
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import lsst.utils
+import lsst.utils.tests
+
+noEotestMsg = ""
+noEotest = False
+try:
+    import lsst.eotest
+except ImportError:
+    noEotestMsg = "No eotest setup, so skipping unit test"
+    noEotest = True
+
+
+class CpPipeTestCase(lsst.utils.tests.TestCase):
+    """A test case for cp_pipe."""
+
+    def testExample(self):
+        pass
+
+    @unittest.skipIf(noEotest, noEotestMsg)
+    def testImport(self):
+        import lsst.cp.pipe as cpPipe
+
+    @unittest.skipIf(noEotest, noEotestMsg)
+    def testClassInstantiation(self):
```

Comment:
You test for instantiation, but don't actually run anything? I guess that's better than nothing, but not by much.

Reply:
Yeah, I know, it's just really hard to run anything without a ton of test data, so I spoke to some locals and we thought that at least setting things up was better than nothing.